ImisRestApi/Escape/Payment/Controllers/PaymentController.cs

```diff
@@ -198,7 +198,7 @@ public override IActionResult PostReqControlNumberAck([FromBody] Acknowledgement
         [Route("api/GetPaymentData")]
         public async Task<IActionResult> GetPaymentChf([FromBody] GepgPaymentMessage model)
         {
-            if (imisPayment.IsValidCall(model, 0))
+            if (imisPayment.IsValidCall(model, 1))
```

Comment:
Please define a constant somewhere and use it here. Having a number is difficult to understand and debug.